From 6fb5be37702e5590395558e7655cdb8bea1fbd59 Mon Sep 17 00:00:00 2001
From: Paul Mundt <lethal@linux-sh.org>
Date: Tue, 17 Jul 2012 15:21:47 +0900
Subject: sh: pfc: Export pinctrl binding init symbol.

symbol_request() requires the registration symbol to be exported, make
sure it is.

Signed-off-by: Paul Mundt <lethal@linux-sh.org>
(cherry picked from commit 1acbbb4ed846ec6a546834c86a3bd4a5bad4ee4d)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/sh/pfc/pinctrl.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/sh/pfc/pinctrl.c b/drivers/sh/pfc/pinctrl.c
index fe1750e..4eaf24b 100644
--- a/drivers/sh/pfc/pinctrl.c
+++ b/drivers/sh/pfc/pinctrl.c
@@ -283,6 +283,7 @@ int sh_pfc_register_pinctrl(struct sh_pfc *pfc)
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(sh_pfc_register_pinctrl);
 
 static inline void __devinit sh_pfc_map_one_gpio(struct sh_pfc *pfc,
 						 struct sh_pfc_pinctrl *pmx,
-- 
1.7.10.4

